ui/lib/kmip/addon/routes/configuration.js

```diff
@@ -1,11 +1,13 @@
 import Route from '@ember/routing/route';
 import { inject as service } from '@ember/service';
+import UnloadModel from 'vault/mixins/unload-model-route';
 
-export default Route.extend({
+export default Route.extend(UnloadModel, {
   store: service(),
   secretMountPath: service(),
   pathHelp: service(),
-  beforeModel() {
+  async beforeModel() {
+    await this.store.unloadAll('kmip/config');
```

Comment:
We don't need this line AND extend UnloadModel, right?

Reply:
Ah yes, you're correct.  I'll remove